python/Ganga/new_tests/TestProxy.py

```diff
@@ -69,9 +70,14 @@ def test_dict_attributes(self):
 
         # Proxy class instance
         self.assertFalse(hasattr(self.p, '_proxyClass'))
-        self.assertFalse(hasattr(self.p, '_proxyObject'))
+        self.assertTrue(hasattr(self.p, '_proxyObject'))
         self.assertTrue(hasattr(self.p, '_impl'))
 
+        # Proxied class
+        self.assertTrue(hasattr(self.p_class, '_proxyClass'))
+        self.assertFalse(hasattr(self.p_class, '_proxyObject'))
+        self.assertFalse(hasattr(self.p_class, '_impl'))
+
```

Comment:
Sorry, I got a bit confused. I thought `self.p_class` was a proxy class but in fact it's a standard `GangaObject` since `addProxy` does nothing to class objects? I was confused by the odd name and the unneccesary use of `addProxy` in the `setUp()`.

In that case these new lines are just duplicating
```python
# Proxied GangaObject class
self.assertTrue(hasattr(TestGangaObject, '_proxyClass'))
self.assertFalse(hasattr(TestGangaObject, '_proxyObject'))
self.assertFalse(hasattr(TestGangaObject, '_impl'))
```